### PR TITLE
2016-06 OME-XML samples: final fixes

### PR DESCRIPTION
**Pull request by sbesson**
This PR makes a couple of final minor adjustments to the schema and samples:

- enforces UTF-8 encoding into the xml header of the sample files as per /https://trello.com/c/oICUieGO/98-file-formats-xml-restrictions
- removes unnecessary prefixes in `hcs.ome.xml` now all schema namespaces are unified
- removes author/written by information from the 2016-06 xsd schema
- ports the the `instrument.ome.xml` extensions from https://github.com/openmicroscopy/bioformats/pull/2427#issuecomment-223998659 to include map examples in `GenericExcitationSource` and `ImagingEnvironment`

### CODE REVIEW

File: components/specification/released-schema/2016-06/ome.xsd [hunk 1]
```diff
@@ -19,11 +19,6 @@
 #       http://www.openmicroscopy.org/info/attribution
 #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 -->
-<!--
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# Written by: Ilya G. Goldberg, Andrew J Patterson
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
```
Comment (hflynn) [inline components/specification/released-schema/2016-06/ome.xsd]: 👍

**Comment by hflynn** [comment]
As discussed in standup, I'd favour sticking with version 1 as we haven't announced this yet or published the linked docs.

**Comment by dgault** [comment]
I have no preference on the versioning but the changes to the samples all look good

**Comment by sbesson** [comment]
See https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge-superbuild/262/ and  https://ci.openmicroscopy.org/view/Failing/job/OME-FILES-CPP-DEV-merge-win-superbuild/287/ for the C++ validation.

**Comment by rleigh-dundee** [comment]
The changes and additions all look good to me.

**Pull request merged**